server/src/internalClusterTest/java/org/elasticsearch/cluster/ClusterHealthIT.java

```diff
@@ -258,18 +258,14 @@ public void testHealthWithClosedIndices() {
 
     public void testHealthOnIndexCreation() throws Exception {
         final AtomicBoolean finished = new AtomicBoolean(false);
-        Thread clusterHealthThread = new Thread() {
-            @Override
-            public void run() {
-                while (finished.get() == false) {
-                    ClusterHealthResponse health = client().admin().cluster().prepareHealth().get();
-                    assertThat(health.getStatus(), not(equalTo(ClusterHealthStatus.RED)));
-                }
+        Thread clusterHealthThread = new Thread(() -> {
```

Comment:
Are these changes meaningful or just cosmetic? If they're not important here then it'd be best to keep them off the feature branch. It'd be fine to make them in `master` tho.